src/utils/input/index.js

```diff
@@ -0,0 +1,81 @@
+/*
+ * LiskHQ/lisky
+ * Copyright © 2017 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import {
+	getStdIn,
+	getPassphrase,
+	getData,
+} from './utils';
+
+export const getFirstLineFromString = multilineString => (
+	typeof multilineString === 'string'
+		? multilineString.split(/[\r\n]+/)[0]
+		: null
+);
+
+const getInputsFromSources = async (vorpal, {
+	passphrase: passphraseInput,
+	secondPassphrase: secondPassphraseInput,
+	password: passwordInput,
+	data: dataInput,
+}) => {
+	const [
+		passphraseIsRequired,
+		secondPassphraseIsRequired,
+		passwordIsRequired,
+		dataIsRequired,
+	] = [passphraseInput, secondPassphraseInput, passwordInput, dataInput]
+		.map(input => !!input && input.source === 'stdin');
+
+	const stdIn = await getStdIn({
+		passphraseIsRequired,
+		secondPassphraseIsRequired,
+		passwordIsRequired,
+		dataIsRequired,
+	});
+
+	const passphrase = typeof stdIn.passphrase !== 'string' && passphraseInput
+		? await getPassphrase(vorpal, passphraseInput.source, {
+			shouldRepeat: passphraseInput.repeatPrompt,
+		})
+		: stdIn.passphrase || null;
+
+	const secondPassphrase = typeof stdIn.secondPassphrase !== 'string' && secondPassphraseInput
+		? await getPassphrase(vorpal, secondPassphraseInput.source, {
+			displayName: 'your second secret passphrase',
+			shouldRepeat: secondPassphraseInput.repeatPrompt,
+		})
+		: stdIn.secondPassphrase || null;
+
+	const password = typeof stdIn.password !== 'string' && passwordInput
+		? await getPassphrase(vorpal, passwordInput.source, {
```

Comment:
Maybe not relevant for this PR. But since we now distinguish between passphrase, password and data, it would make sense to have a function `getPassword` like `getData`. One benefit I could think of is verifying passphrases to have 12 words, which are not required for passwords.

Reply:
I like the idea, but we'd need a `--no-verify` flag so people could use other forms of passphrase if they need

Reply:
yes, sounds good

Reply:
https://github.com/LiskHQ/lisky/issues/337